eq-author/src/enhancers/withValidationError.js

```diff
@@ -0,0 +1,50 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+import VALIDATION_MESSAGES from "constants/validationMessages";
+
+const withValidationError = entityPropName => WrappedComponent => {
+  return class extends React.Component {
+    static propTypes = {
+      enableValidationMessage: PropTypes.bool,
+      [entityPropName]: PropTypes.object.isRequired, // eslint-disable-line
+    };
+
+    static defaultProps = {
+      enableValidationMessage: true,
+    };
+
+    getValidationError = ({ field, label }) => {
+      const {
+        [entityPropName]: { validationErrorInfo },
+      } = this.props;
+
+      const messages = validationErrorInfo && validationErrorInfo.errors;
+
+      if (!messages || this.props[entityPropName].isNew) {
```

Comment:
Minor nit: Could we extract `isNew` on line `19` as well?

Reply:
done